### PR TITLE
New Hair and Neck Bows and Ribbons

### PR DESCRIPTION
**Pull request by SleepyGemmy**
this PR adds an alternative hair bow, a neck bow, and a neck ribbon. it also makes these and the bowtie into contained sprites.

### CODE REVIEW

**Comment by KingOfThePing** [comment]
anime larp

**Review by KingOfThePing — CHANGES_REQUESTED** [review]

File: code/modules/clothing/head/misc.dm [hunk 1]
```diff
@@ -80,17 +80,29 @@
 	desc = "A Silversun dawnflower pin, named after the same flower. This particular version is an artificial recreation, and lacks the distinctive bioluminescence of the original."
 
 /obj/item/clothing/head/pin/bow
+	name = "bow"
+	desc = "You aren't supposed to see this."
```
[the remark this answers is not in the document]
Comment (SleepyGemmy) [reply code/modules/clothing/head/misc.dm:84]: parent item.

**Comment by SleepyGemmy** [comment]
!review

**Review by Arrow768 — APPROVED** [review]

**Comment by SleepyGemmy** [comment]
PR is waiting for reviews, it is **NOT ABANDONED**.

**Review by JohnWildkins — APPROVED** [review]

**Pull request merged**